.travis.yml

```diff
@@ -36,7 +52,7 @@ addons:
 
 install:
   - |
-    if [ $TRAVIS_PHP_VERSION != '5.6' ]; then
+    if [[ $TRAVIS_PHP_VERSION != '5.6' || $TRAVIS_PHP_VERSION != hhv* ]]; then
```

Comment:
xdebug is not enabled by default on hhvm, additionally phpenv for setting environment variables [has issues with hhvm](https://github.com/travis-ci/travis-ci/issues/3930). The correct way to disable xdebug on travis for hhvm is
```php
# Disable xdebug in hhvm.
if [[ $TRAVIS_PHP_VERSION = hhv* ]]; then echo 'xdebug.enable = 0' >> /etc/hhvm/php.ini; fi
```
but it is unneccesary since xdebug is not enabled by default on hhvm